packages/ui-library/src/components/icon/index.ts

```diff
@@ -20,13 +28,8 @@ export class BlrIcon extends LitElement {
   @property() theme: ThemeType = 'Light';
   @property() classMap?: DirectiveResult<typeof ClassMapDirective>;
 
-  // these are not triggered directly but allows us to map it internally and bve typesafe
-  @property() blrClick?: () => void;
-
-  protected handleClick = (event: Event) => {
-    this.dispatchEvent(
-      new CustomEvent('blrClick', { bubbles: true, composed: true, detail: { originalEvent: event } })
-    );
+  protected handleClick = (event: MouseEvent | KeyboardEvent) => {
+    this.dispatchEvent(createBlrClickEvent({ originalEvent: event }));
```

Comment:
Curly bracket can be removed

Reply:
It probably shouldn't be - `this.dispatchEvent()` returns a boolean, if I remove the brackets `handleClick` returns a boolean too, which doesn't make sense and could theoretically have an unexpected effect where handleClick is used.

Reply:
Aah, true, make sense, thanks for the insight.